### DISCUSSION TITLE
No downloads in Firefox about:downloads tab

### DISCUSSION THREAD

**Discussion by Mocha724** [aclap-dev/video-downloadhelper#181] [Help and Q&A]
### Your question

Hi!

I want the the list of actual downloads and the history of downloded files back in the Firefox about:downloads tab. Is that possible? Its standard on VDH 8.1.4.0, but not with VDH 8.2.0.23 

![DownloadHelper Test](https://github.com/aclap-dev/video-downloadhelper/assets/160655914/7e1c4abf-0f56-4def-9306-aaf6fb1a5494)

I don't like it only here:

![DownloadHelper Test2](https://github.com/aclap-dev/video-downloadhelper/assets/160655914/2b0b5408-8d16-44ff-8632-02de6c89e409)

Thanks,
Mocha

### Addon & CoApp version and details

Firefox 123
VDH 8.2.0.23
CoApp 2.0.19

**Comment by paulrouget** [comment]
Fixed in version 8.2.1. Please wait until Mozilla updates the addon.
